gcloud-java-dns/src/main/java/com/google/gcloud/dns/RecordSet.java

```diff
@@ -276,13 +276,15 @@ public Type type() {
   }
 
   @Override
-  public int hashCode() {
+  public final int hashCode() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary final modifier in final class](https://www.codacy.com/app/mziccard/gcloud-java/file/2392420333/issues/source?bid=3123103&fileBranchId=3228372#l279)